Caching Performance Fix

=svnm= https://source.sakaiproject.org/svn/db/branches/SAK-13886/@48283 db 

Not in 2.5.1, just in trunk presently 
https://source.sakaiproject.org/svn/portal/branches/SAK-13886/@48383 portal 

--- /dev/null
+++ SAK-13886.tmp
@@ -0,0 +1 @@
+ 
